app/components/pipeline/modal/search-event/styles.scss

```
@@ -7,24 +7,42 @@
       .modal-dialog {
         max-width: 30rem;
 
+        #search-select {
+          display: flex;
+          width: 8rem;
+          float: left;
+          overflow: hidden;
+          white-space: nowrap;
+
+          select {
+            display: flex;
+            align-items: center;
+            padding: 0.375rem 0.75rem;
+            margin-bottom: 0;
+          }
+        }
+
```

```suggestion
select {
  width: 7rem;
  margin-right: 1rem;
}
```
Move this into the `#search-input` section of the CSS